### DISCUSSION TITLE
Update `statsforecast` to version > 1.4

### DISCUSSION THREAD

**Discussion by d-a-bunin** [etna-team/etna#416] [Improvements]
### Description

Currently, we have a version of `statsforecast=1.4.0`.

We can't update it higher due to fixed version of `antlr4-python3-runtime` in `omegaconf` dependency. The [latest news about updating `omegaconf`](https://github.com/omry/omegaconf/pull/1114).

After the change we could remove implementation of `ARIMA` from `etna.libs`.

### Thoughts

This change can be potentially a breaking change, but I'm unsure.

**Comment by egoriyaa** [comment]
Solved in #599
